runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/helpers/elementForm.ts

```diff
@@ -78,21 +85,36 @@ export const getFormSchema = (
     let selectedAction = action;
     if (!action) selectedAction = getActionFromElement(selectedElement);
 
+    const hasOutput = selectedAction.form.schema.properties.output;
+
+    const additionalProperties = {
+        disabled: {
+            type: 'boolean',
+            title: translate(
+                'Process.Details.Modeler.ActionPanel.Form.Disabled.Title'
+            )
+        },
+        runFromHere: {
+            type: 'boolean',
+            title: translate(
+                'Process.Details.Modeler.ActionPanel.Form.RunFromHere.Title'
+            )
+        },
+    } as JSONSchema7['properties'];
```

Comment:
use `satisfies` or `: JSONSchema7['properties']` at the beginning. Avoid using `as` if possible